gapis/api/gles/custom_replay.go

```diff
@@ -312,6 +312,21 @@ func (ω *EglCreateImageKHR) Mutate(ctx context.Context, id api.CmdID, s *api.Gl
 	return cb.ReplayCreateExternalImage(ctxID, TextureId(ω.Buffer().Address()), ω.Result()).Mutate(ctx, id, s, b, nil)
 }
 
+func (ω *EglSwapBuffers) Mutate(ctx context.Context, id api.CmdID, s *api.GlobalState, b *builder.Builder, w api.StateWatcher) error {
+	err := ω.mutate(ctx, id, s, nil, w)
+	if b == nil || err != nil {
+		return err
+	}
+	// Get context ID from current Thread
+	context := GetState(s).Contexts().Get(ω.Thread())
+	if context.IsNil() {
+		return fmt.Errorf("No EGL context in thread calling eglSwapBuffers, thread: %v", ω.Thread())
```

Comment:
I think we should just ignore it here instead of error-ing out.

Reply:
Ignoring it silently may break things in subtle ways down the line (basically eglSwapBuffers() won't be replayed, with no hint of why), so I am more on the side of error-ing it as soon as possible.